src/client/VZSidebar/styles.scss

```
@@ -22,6 +22,7 @@
   }
 
   .file-or-directory {
+    height: max(30px);
```

In this context, max(30px) effectively just sets the height to 30px, because there is no other value to compare it against. Therefore, it can be simplified by directly setting the height to 30px without using the max() function.

```suggestion
    height: 30px;
```